PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -113,20 +116,17 @@ internal class PayPalInternalClient(
         callback: PayPalInternalClientCallback
     ) {
         braintreeClient.sendPOST(
-            url = url,
-            data = requestBody,
+            url = url, data = requestBody,
         ) { responseBody: String?, httpError: Exception? ->
-
             if (responseBody == null) {
                 callback.onResult(null, httpError)
                 return@sendPOST
             }
-
             try {
-                val paypalPaymentResource = fromJson(responseBody)
+                val paypalPaymentResource = PayPalPaymentResource.fromJson(responseBody)
                 val parsedRedirectUri = Uri.parse(paypalPaymentResource.redirectUrl)
-
                 val pairingId = findPairingId(parsedRedirectUri)
+                payPalSetPaymentTokenUseCase.setPaymentToken(pairingId)
```

Comment:
Take it or leave it - we don't use the phrase "pairingID" within our PayPal module on iOS. IMO it's a little confusing to call them different things across the platforms, since I didn't know what "pairingID" meant when first reading the PR. 

[This](https://github.com/braintree/braintree_ios/blob/d33df60b4ca041d19a9f6888423abc9de4350516/Sources/BraintreePayPal/BTPayPalClient.swift#L392) is sort of the equivalent of the `findPairingId()` function on iOS.

Maybe we can align on calling them `paypalContextId` across both platforms instead. Food for thought / maybe a clean-up PR.